Link checker GitHub Action fails with "muffet.sh: command not found"

The [weekly link checker action](https://github.com/medic/cht-docs/commit/6f0f10bad37a10ea298a0a34a363e08f2e9625b6/checks) failed with:

>Run cd .github/scripts/
/home/runner/work/_temp/34cf8ccf-5224-40db-8f1c-b[9](https://github.com/medic/cht-docs/commit/6f0f10bad37a10ea298a0a34a363e08f2e9625b6/checks#step:7:10)b330254db7.sh: line 3: muffet.sh: command not found
Error: Process completed with exit code 127.

This might be caused by https://github.com/medic/cht-docs/pull/1733, mainly the changes to [.github/workflows/cron-check-links.yml](https://github.com/medic/cht-docs/pull/1733/files#diff-02e8e66e2fdde3ba990efcad4057962bd3d6da9294591c3dae57d015d4af93ad):

![Image](https://github.com/user-attachments/assets/3e90902b-9d71-4e77-ba16-d337eeb773ec)

--- a/.github/workflows/cron-check-links.yml
+++ b/.github/workflows/cron-check-links.yml
@@ -43,7 +43,8 @@ jobs:
         shell: bash
         run:  |
           chmod +x ./.github/scripts/*muffet*
-          ./.github/scripts/muffet.sh
+          cd .github/scripts
+          ./muffet.sh
 
       - name: Report errors to Slack, if any
         if: ${{ github.ref == 'refs/heads/main' && failure() }}

--- a/.github/workflows/cron-check-links.yml
+++ b/.github/workflows/cron-check-links.yml
@@ -42,9 +42,8 @@ jobs:
       - name: Run Muffet link checker
         shell: bash
         run:  |
-          cd .github/scripts/
-          chmod +x *muffet*
-          muffet.sh
+          chmod +x ./.github/scripts/*muffet*
+          ./.github/scripts/muffet.sh
 
       - name: Report errors to Slack, if any
         if: ${{ github.ref == 'refs/heads/main' && failure() }}

